### PR TITLE
Added Docker build, fixed gravity

### PR DESCRIPTION
**Pull request by davetcoleman**
Reinstate gravity in correct location
Reduce verbosity of various messages

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# RethinkRobotics/baxter-simulator:kinetic
```
Comment (davetcoleman) [inline Dockerfile:1]: A running example is here: https://hub.docker.com/r/davetcoleman/baxter_simulator/builds/

File: baxter_gazebo/worlds/baxter.world [hunk 2]
```diff
@@ -11,11 +11,12 @@
...
     <physics type="ode">
       <real_time_update_rate>1000.0</real_time_update_rate>
-      <gravity>
-	0.0 0.0 -9.81
-      </gravity>
     </physics>
 
+    <gravity>
+      0.0 0.0 -9.81
+    </gravity>
```
Comment (davetcoleman) [inline baxter_gazebo/worlds/baxter.world:18]: gravity is now a member of world, not physics

http://sdformat.org/spec?ver=1.6&elem=world#world_gravity

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ENV TERM xterm
+
+# Setup catkin workspace
+ENV CATKIN_WS=/root/ws_baxter
+RUN mkdir -p $CATKIN_WS/src
+WORKDIR $CATKIN_WS/src
+
+# Download source code
+RUN wstool init . && \
+    wstool merge https://raw.githubusercontent.com/RethinkRobotics/baxter_simulator/${ROS_DISTRO}-gazebo7/baxter_simulator.rosinstall && \
```
Comment (IanTheEngineer) [inline Dockerfile]: Currently, this is the link:
https://raw.githubusercontent.com/RethinkRobotics/baxter_simulator/kinetic-devel/baxter_simulator.rosinstall

File: baxter_gazebo/worlds/baxter.world (hunk 2 above)
Comment (IanTheEngineer) [reply baxter_gazebo/worlds/baxter.world:18]: Oh wow. Good catch...

File: Dockerfile (hunk 1 above)
Comment (IanTheEngineer) [reply Dockerfile:1]: Do you have a link or tutorial that helped you set this up? I'd like to do it for the whole SDK...

**Review by IanTheEngineer — APPROVED** [review]
Looks good, thanks for the PR! Please fix the `wstool` URL and I'll merge

File: baxter_gazebo/worlds/baxter.world (hunk 2 above)
Comment (davetcoleman) [reply baxter_gazebo/worlds/baxter.world:18]: for indigo it was correct, they changed it for gazebo7

File: Dockerfile (hunk 1 above)
Comment (davetcoleman) [reply Dockerfile:1]: To learn docker I recommend Docker's basic tutorials, or you could skip that and go straight to the [ROS](http://wiki.ros.org/docker/Tutorials/Docker) ones or [moveit](http://moveit.ros.org/install/docker/) ones

**Comment by davetcoleman** [comment]
Fixed

**Review by IanTheEngineer — APPROVED** [review]
Looks great! Thanks for the PR, Dave.

**Pull request merged**